R/pkg/tests/run-all.R

```diff
@@ -38,6 +38,10 @@ sparkRFilesBefore <- list.files(path = sparkRDir, all.files = TRUE)
 sparkRTestMaster <- "local[1]"
 if (identical(Sys.getenv("NOT_CRAN"), "true")) {
   sparkRTestMaster <- ""
+} else {
+  # Disable hsperfdata on CRAN
+  old_java_opt <- Sys.getenv("_JAVA_OPTIONS")
+  Sys.setenv("_JAVA_OPTIONS"=paste("-XX:-UsePerfData", old_java_opt, sep=" "))
```

Comment:
I think perhaps it'd better to set `spark.driver.extraJavaOptions` and `spark.executor.extraJavaOptions` (use a variable to store its value, as in `sparkRTestMaster`, or like in vignettes, `sparkSessionConfig`)

btw, I think the convention is camelCase variables :)

Reply:
also `sep=" "` is the default

Reply:
space between variable/value
`"_JAVA_OPTIONS" = paste("-XX:-UsePerfData", old_java_opt, sep = " "`
we *should* have a lintr rule that detects this at one point..